qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulation.cpp

```diff
@@ -62,8 +62,8 @@ void InelasticDataManipulation::initLayout() {
   addMVPTab<InelasticDataManipulationSymmetriseTab, InelasticDataManipulationSymmetriseTabView>("Symmetrise");
   addMVPTab<InelasticDataManipulationSqwTab, InelasticDataManipulationSqwTabView>("S(Q, w)");
   addMVPTab<InelasticDataManipulationMomentsTab, InelasticDataManipulationMomentsTabView>("Moments");
+  addMVPTab<InelasticDataManipulationIqtTab, InelasticDataManipulationIqtTabView>("Iqt");
   addMVPTab<InelasticDataManipulationElwinTab, InelasticDataManipulationElwinTabView>("Elwin");
-  addTab<InelasticDataManipulationIqtTab>("Iqt");
```

Comment:
Does the order of the tabs change because the Iqt tab is now added before Elwin?

Reply:
I have tried and it does not seem to mind the order here. 
I think that's done in 'enabledtabs' QstringList.
Anyway I have changed it for better readability.